CHANGELOG.md

```diff
@@ -689,15 +689,14 @@ In addition to new features and major improvements, Rocket saw the following imp
 ## Infrastructure
 
   * CI was moved from Travis to Azure Pipelines; Windows support is tested.
-  * Rocket's chat moved to [Matrix] and [Freenode].
```

Comment:
We should never change the past. Please revert this change.

Reply:
I don't agree that this is changing the past.  But it's your project, so I have reverted that change, as you request.

Reply:
This is a CHANGELOG entry for 0.4.4, where Rocket's chat moved to Freenode. Changing it to say we moved to Libera is plain wrong as a 0.4.4 entry.